### PR TITLE
Clarify relationship between Ord and Eq by updating the antisymmetry law

### PR DESCRIPTION
**Pull request by JamieBallingall**
Change "=" to "==" in antisymmetry law comment to better show the connection between Ord and Eq. Closes issue #174.
---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [X] Linked any existing issues or proposals that this pull request should close
- [X] Updated or added relevant documentation
- [NA] Added a test for the contribution (if applicable)

### CODE REVIEW

**Review by JordanMartinez — APPROVED** [review]

**Comment by JamieBallingall** [comment]
Thanks @JordanMartinez. Is there anything else I need to do on my side before this gets merged or are we waiting on an additional approval or some release milestone or something?

**Comment by JordanMartinez** [comment]
Since this isn't breaking, just another approval.

**Review by natefaubion — APPROVED** [review]

**Pull request merged**
